client/packages/system/src/Stock/api/hooks/utils/useStockApi.ts

```diff
@@ -1,10 +1,29 @@
-import { useAuthContext, useGql } from '@openmsupply-client/common';
-import { getSdk } from '../../operations.generated';
+import {
+  FilterBy,
+  SortBy,
+  useAuthContext,
+  useGql,
+} from '@openmsupply-client/common';
+import { getSdk, StockLineRowFragment } from '../../operations.generated';
 import { getStockQueries } from '../../api';
 
+export type ListParams = {
+  first: number;
+  offset: number;
+  sortBy: SortBy<StockLineRowFragment>;
+  filterBy: FilterBy | null;
+};
+
 export const useStockApi = () => {
```

Comment:
Is useStockLine a better name? its obvious once you think about it but since we usually talk about stock lines it might the better name?

Reply:
it is - I've been wondering the same!